x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/details_page/components/package_policies/index.tsx

```diff
@@ -16,14 +16,14 @@ import { PackagePoliciesTable } from './package_policies_table';
 export const PackagePoliciesView = memo<{ agentPolicy: AgentPolicy }>(({ agentPolicy }) => {
   useBreadcrumbs('policy_details', { policyName: agentPolicy.name });
 
-  if (agentPolicy.package_policies.length === 0) {
+  if (!agentPolicy.package_policies || agentPolicy.package_policies.length === 0) {
```

Comment:
nit: there is a shorter way with lodash: `isEmpty(agentPolicy.package_policies)`